### PR TITLE
modifying the multipath_test.py script for scenario testing

### PR DESCRIPTION
**Pull request by Naresh-ibm**
converted the code into functions, based on different scenarios,
So that each scenario can be ran when needed.
script also works fine as a whole.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[root@ltczz401-lp4 ~]# avocado run multipath_test.py -m multipath_test.yaml
Error running method "pre_tests" of plugin "fetchasset": 'Attribute' object has no attribute 'id'
JOB ID     : 7f33363dd6049e2cef9a58d5c87c3b8f3f3c6d9c
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-22T02.02-7f33363/job.log
 (01/21) multipath_test.py:MultipathTest.test;queue-length-3e82: PASS (128.33 s)
 (02/21) multipath_test.py:MultipathTest.test;service-time-e6a8: PASS (126.74 s)
 (03/21) multipath_test.py:MultipathTest.test;round-robin-ba3e: PASS (126.18 s)
 (04/21) multipath_test.py:MultipathTest.test_fail_reinstate_individual_paths;queue-length-3e82: PASS (37.74 s)
 (05/21) multipath_test.py:MultipathTest.test_fail_reinstate_individual_paths;service-time-e6a8: PASS (37.87 s)
 (06/21) multipath_test.py:MultipathTest.test_fail_reinstate_individual_paths;round-robin-ba3e: PASS (38.84 s)
 (07/21) multipath_test.py:MultipathTest.test_io_run_on_single_path;queue-length-3e82: PASS (393.05 s)
 (08/21) multipath_test.py:MultipathTest.test_io_run_on_single_path;service-time-e6a8: PASS (395.42 s)
 (09/21) multipath_test.py:MultipathTest.test_io_run_on_single_path;round-robin-ba3e: PASS (389.99 s)
 (10/21) multipath_test.py:MultipathTest.test_failing_reinstate_all_paths;queue-length-3e82: PASS (404.64 s)
 (11/21) multipath_test.py:MultipathTest.test_failing_reinstate_all_paths;service-time-e6a8: PASS (404.55 s)
 (12/21) multipath_test.py:MultipathTest.test_failing_reinstate_all_paths;round-robin-ba3e: PASS (405.13 s)
 (13/21) multipath_test.py:MultipathTest.test_removing_all_paths;queue-length-3e82: PASS (369.67 s)
 (14/21) multipath_test.py:MultipathTest.test_removing_all_paths;service-time-e6a8: PASS (369.89 s)
 (15/21) multipath_test.py:MultipathTest.test_removing_all_paths;round-robin-ba3e: PASS (369.83 s)
 (16/21) multipath_test.py:MultipathTest.test_suspend_resume_individual_mpath;queue-length-3e82: PASS (363.46 s)
 (17/21) multipath_test.py:MultipathTest.test_suspend_resume_individual_mpath;service-time-e6a8: PASS (363.60 s)
 (18/21) multipath_test.py:MultipathTest.test_suspend_resume_individual_mpath;round-robin-ba3e: PASS (363.44 s)
 (19/21) multipath_test.py:MultipathTest.test_suspend_resume_all_mpath;queue-length-3e82: PASS (182.69 s)
 (20/21) multipath_test.py:MultipathTest.test_suspend_resume_all_mpath;service-time-e6a8: PASS (182.65 s)
 (21/21) multipath_test.py:MultipathTest.test_suspend_resume_all_mpath;round-robin-ba3e: PASS (182.50 s)
RESULTS    : PASS 21 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 5636.99 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-22T02.02-7f33363/results.html
[root@ltczz401-lp4 ~]#

**Comment by Naresh-ibm** [comment]
[multipath_scenario_py3_v4_job_log.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3879024/multipath_scenario_py3_v4_job_log.txt)

File: io/disk/multipath_test.py [hunk 1]
```diff
@@ -151,46 +153,150 @@ def test(self):
...
+        time.sleep(self.op_long_sleep_time)
+
+        for dic_mpath in self.mpath_list:
+            if multipath.resume_mpath(dic_mpath["name"]) is False:
+                self.log.info("couldn't resume %s ", dic_mpath["name"])
+                err_mpaths.append(dic_mpath['name'])
+        if err_mpaths:
+            self.fail("error mpaths in suspnd all mpaths: %s" % err_mpaths)
+
+    def remove_add_mpath(self):
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: @Naresh-ibm I think this needs to be `test_remove_add_mpath`. Once this is changed, we are ready to merge.

**Comment by Naresh-ibm** [comment]
closing this in favor of #1238

**Comment by narasimhan-v** [comment]
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1438

**Pull request closed without merging**